src/main/resources/templates/signup.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <section th:replace="fragments/head :: headBootstrap"></section>
+        <meta charset="UTF-8">
+        <title>Sign\Up</title>
+    </head>
+    <body>
+        <h1>Sign\Up</h1>
+        <form action="/signup" method="post">
+            <label for="username" style="size:1px ; visibility: hidden">username</label><input type="text" name="username" placeholder="Username" id="username">
+            <label for="password" style="size:1px ; visibility: hidden">password</label><input type="text" name="password" placeholder="Password" id="password">

```

Comment:
For the <input> tag on your password, give it a type attribute of password... that will hide what they type.